src/pages/docs/tutorials/tutorial-typescript-enact/adding-a-new-component/index.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
There isn’t really any TypeScript happening in this page or example, other than using the `.tsx` file extension.  We might need to change the page title to something like `Making a Counter Class With JavaScript`, then last page could be `Converting the Counter Class to Enact Component Using TypeScript`…maybe?

Also, I wonder if we should (not for this ticket) add a “copy to clipboard” button for the code examples (and then add them to other tutorials/examples)?